pandas/core/groupby/grouper.py

```diff
@@ -31,6 +32,17 @@
 
 from pandas.io.formats.printing import pprint_thing
 
+_observed_msg = textwrap.dedent(
+    """\
+Using 'observed=False', because grouping on a categorical. A future version
+of pandas will change to 'observed=True'.
```

Comment:
This message here is not what I understood from the discussion in the PR? (https://github.com/pandas-dev/pandas/pull/35967#issuecomment-693685382)

Reply:
Yeah, I thought the suggestion was that this would *raise* in the future. So this warning message would be something like

```
Grouping by a categorical, but 'observed' was not specified.
Using 'observed=False', but in the future this will will raise
a ValueError. Pass 'observed=True' or 'observed=False' to
silence this warning.
```

Maybe with a link to the docs on what observed=True/False means.

Reply:
Updated